tabpy-server/tabpy_server/common/endpoint_file_mgr.py

```diff
@@ -13,25 +13,28 @@
 import shutil
 from re import compile as _compile
 
-logger = logging.getLogger(__name__)
 
 _name_checker = _compile(r'^[a-zA-Z0-9-_\s]+$')
 
 
-def _check_endpoint_name(name):
+def _check_endpoint_name(name, logger=logging.getLogger(__name__)):
```

Comment:
Not sure the intention of making this a keyword argument but probably better to just initialize the logger in the global namespace of your module rather than defaulting it locally to this function

Reply:
Depending on the context the function can be called from HTTP request handler and then `logger` will be a wrapper which appends call ID to a logged message. Or it can be called from an app (e.g. when deploying a model) and then it uses the standard logger.